java/core/src/main/java/sleeper/core/statestore/AllReferencesToAFileSerDe.java

```diff
@@ -67,18 +67,17 @@ public AllReferencesToAFile deserialize(JsonElement json, Type type, JsonDeseria
             }
             return AllReferencesToAFile.builder()
                     .filename(filename)
-                    .totalReferenceCount(object.get("totalReferenceCount").getAsInt())
-                    .internalReferences(references)
+                    .references(references)
                     .build();
         }
 
         @Override
         public JsonElement serialize(AllReferencesToAFile file, Type type, JsonSerializationContext context) {
             JsonObject object = new JsonObject();
             object.addProperty("filename", file.getFilename());
-            object.addProperty("totalReferenceCount", file.getTotalReferenceCount());
+            object.addProperty("totalReferenceCount", file.getReferenceCount());
```

Comment:
It looks like we don't need to write this field any more, since we're not reading it.